src/pylero/document.py

```diff
@@ -454,7 +452,12 @@ def get_work_items(
             ).uri
         else:
             parent_uri = None
-        p_fields = _WorkItem._convert_obj_fields_to_polarion(fields)
+        p_fields = []
+        for wi in _SpecificWorkItem.__subclasses__():
```

Comment:
- we never import _SpecificWorkItem, it's just used for creating new 
types
- I see the same flow is being used in many classes, like filling `p_fields` without getting custom fields fields
- If anything needs to be fixed, it should probably go into `BasePolarion`

Reply:
> * we never import _SpecificWorkItem, it's just used for creating new
>   types
okay, how should i then grab all the new types created for my loop?
> * I see the same flow is being used in many classes, like filling `p_fields` without getting custom fields fields
> * If anything needs to be fixed, it should probably go into `BasePolarion`
okay, then let me see, if I can move the whole thing into `BasePolarion`!

Reply:
Would it be okay to import `woritems` from line 1583 in the module `work_items?https://github.com/RedHatQE/pylero/blob/3af53c70febba00c6268ec8c52beac20969c793c/src/pylero/work_item.py#L1583